src/main/java/banking/CardValidator.java

```diff
@@ -0,0 +1,11 @@
+package banking;
+
+public class CardValidator {
+
+    public boolean isValidCard(CardGenerator generator, String card) {
```

Comment:
Why `isValidCard` method has `CardGenerator` as the input parameter?

Reply:
Can you rename `card ` to the `cardNumber`?

Reply:
because this method need cheksum from CardGenerator

Reply:
Okey. Can you pass the `checksum` instead of the `CardGenerator` object?

Reply:
Yes, I passed the checksum instead of the CardGenerator object in new version of program?